### ISSUE TITLE
Running nocodb without access to github

### ISSUE THREAD

**Issue by maksim77** [nocodb/nocodb#3540]
```
NocoDB used as docker : true
NocoDB version : 0.96.1
Database used in NC_DB URL  : pg
Project was created  by clicking : New Project by connecting to external database
Database on which spreadsheet is created : pg
OS on which NocoDB is running : docker
Node.js version if running as node : 
Database version : 
```

**Steps To Reproduce**

1. Update from 0.92.4 to 0.96.1
2. Open dashboard

When loading the dashboard, a request is made to github.com to find out the latest version of nocodb. But if the service is running in an environment where there is no access to github, we will never see the title page :)

https://github.com/nocodb/nocodb/blob/3d4aa13a2d8c35d1288209747481b102dafacdc9/packages/nocodb/src/lib/meta/api/utilApis.ts#L48-L62

**Comment by o1lab** [comment]
Thank you for notifying about this

**Comment by o1lab** [comment]
Fixed in 0.96.2
